source/includes/k8s/file-transfer-protocol-k8s.rst

```diff
@@ -165,3 +166,73 @@ If SFTP is enabled, the output resembles the following:
 
    enableSFTP: true
 
+.. _minio-certificate-key-file-sftp-k8s
+
+Connect to MinIO Using SFTP with a Certificate Key File
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: RELEASE.2024-05-07T06-41-25Z
+
+
+MinIO supports mutual TLS (mTLS) certificate-based authentication on SFTP, where both the server and the client verify the authenticity of each other.
+
+This type of authentication requires the following public key files:
+
+1. Trusted certificate authority
+2. MinIO Server signed by the certificate authority
```

Comment:
Is this just require MinIO Server to enforce TLS where it's own certs are minted by a the CA from 1?

Reply:
The server has to send this file to the client so the client knows it can trust the server.